core/kazoo_documents/src/kzd_user.erl

```diff
@@ -254,3 +255,10 @@ devices(UserJObj) ->
             lager:warning("unable to find documents owned by ~s: ~p", [UserId, _R]),
             []
     end.
+
+-spec fetch(ne_binary(), api_binary()) -> {'ok', doc()} | {'error', any()}.
+fetch(<<_/binary>> = UserId, <<_/binary>> = Account) ->
```

Comment:
I would prefer `fetch(Account, UserId)` if we put this here. Also, it should be `invalid_parameters` below ;)